apps/condo/domains/common/components/ButtonWithDisabledTooltip.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from '@emotion/styled'
+import React from 'react'
+
+import { Button, ButtonProps, Tooltip } from '@open-condo/ui'
+
+
+type ButtonWithDisabledTooltipProps = ButtonProps & { title: string }
+
+const DisabledButtonWrapper = styled.div<{ disabled }>`
+    ${props => props.disabled ? 'cursor: not-allowed;' : 'cursor: auto;'}
+    .condo-btn {
+      width: 100%;
```

Comment:
Look's little hackky) Why it's not a component behaviour by itself?

Reply:
Ant has a bug that tooltip doesn’t work on the disabled button.
Because it is a hack, I thought it is not worth putting it in the UI-kit